src/main/java/emissary/command/ServerCommand.java

```diff
@@ -36,6 +36,10 @@ public class ServerCommand extends ServiceCommand {
     @Parameter(names = {"--dumpJettyBeans"}, description = "dump all the jetty beans that loaded")
     private boolean dumpJettyBeans = false;
 
+    @Parameter(names = {"--strict"}, description = "If one Place fails to start, shut down the entire server")
```

Comment:
I know the rest of the params don't' do this, but I prefer our param descriptions to indicate what the default is

Reply:
We should follow up and maybe add the default to the other commands